### ISSUE TITLE
[Bug]: Incorrect analysis of named function arguments

### ISSUE THREAD

**Issue by ManuelHentschel** [Code-Inspect/flowr#761]
### What happened?

In the example below, the slice of `print(x)` seems to be identified incorrectly, but for `print(y)` it works.

![image](https://github.com/Code-Inspect/flowr/assets/53863351/45a2088d-bf9a-4dee-bd39-3a88cf99a84f)

### What version of R do you have installed?

4.3.3

### What version of flowR did you use?

1.2.0 (npm install)

### Related input code

```R
x <- 100
list(123, x = 200, 234)
print(x)


y <- 300
list(123, y = 400)
print(y)
```

### Meta Information

- [ ] The Issue is Urgent
- [ ] I Used the Dockerized Version of flowr

**Comment by EagleoutIce** [comment]
Thank you very much for noting this regression caused by me [wrongly handling named arguments](https://github.com/Code-Inspect/flowr/pull/526/commits/fb574142819a171c8bf02c15eadd9a26d2308566#diff-3a9e247156d42987f168306d25bb4d6d998f2f4090a0bec93429150f6892fc82L77-L84) (actually treating them like parameters). I have integrated the patch as part of the complete dataflow overhaul (tracked in #526), the issue should be resolved automatically once i merge it (which should happens soon).

**Comment by EagleoutIce** [comment]
_This issue has been resolved in v2.0.0 (see [Release v2.0.0 (Dataflow v2)](https://github.com/Code-Inspect/flowr/releases/tag/v2.0.0))._
